tools/bundle-size-tools/src/ADO/PrCommentsUtils.ts

```diff
@@ -118,7 +118,7 @@ export class prCommentsUtils {
     await gitApi.updateThread(thread, this.repoId, this.pullRequestId, existingThread.id);
   }
 
-  private async getThreadByType(threadType: string): Promise<GitPullRequestCommentThread> {
+  private async getThreadByType(threadType: string): Promise<GitPullRequestCommentThread | undefined> {
```

Comment:
> | undefined [](http://example.com/codeflow?start=88&length=12)

This function can return undefined and all callers were already handling that possibility.